src/InputForm/InputForm.js

```diff
@@ -0,0 +1,210 @@
+import React from 'react';
+import { NavLink, Route } from 'react-router-dom';
+
+
+// import styling. 
+import './InputForm.scss';
+import '../Header/Header.scss';
+import '../NavBar/Nav.scss';
+
+// import application components
+import Footer from '../Footer/Footer';
+import PaymentAmort from '../PaymentAmort/PaymentAmort';
+import ContactForm from '../ContactForm/ContactForm';
+import Mascot from '../Mascot/Mascot';
+
+
+// images
+import logo from '../images/MA-accumetrix-logo.png';
+
+export default class InputForm extends React.Component { 
+
+    handleClick = (e) => {
+        this.props.analyzeForm(e)
+        this.props.history.push('/ResultsPage')
+    }
+
+    render() {
+    return (
+      <>
+        <header className="header__container">
+            <div className="header__corp">
+                <img className="header__corp-logo" src={logo} alt="company Logo" width='200' />
+                <p className="header__corp-info">License No 12036</p>
+                <p className="header__corp-info2">Independently Owned and Operated Franchise</p>
+            </div>
+                <div className="header__title-container">
+                <h1 className="header__title">Mortgage Rate Estimate</h1>
+            </div>
+            <div className="nav__container">
+                <div className="nav__apply-now-offer">
+                    <a className="nav__apply-now-button" href="http://mortgagealliance.com/JohnHeckbert/apply-online" target="blank" > Apply Now</a>
+                </div>
+                <NavLink className="nav__link" activeClassName="active" to='/Contact Us'>Contact Us</NavLink>
+                <Route path='/Contact Us' component={ContactForm} />
+                <NavLink className="nav__link" activeClassName="active" to='/Mascot'>Mascot</NavLink>
+                <Route path='/Mascot' component={Mascot} />
+                <NavLink className="nav__link" activeClassName="active" to='/Mortgage Rate Estimate'>Mortgage Rate Estimate</NavLink>
+                <Route path='/Mortgage Rate Estimate' component={InputForm} />
+                <NavLink className="nav__link" activeClassName="active" to='/Payment Calculator'>Payment Calculator</NavLink>
+                <Route path='/Payment Calculator' component={PaymentAmort} />
+                <a className="nav__mopolo-link" href="http://mortgagealliance.com/JohnHeckbert/mopolo" target="blank">Credit Monitoring Tool</a>
+            </div>
+        </header>
```

Comment:
I'm not sure what this has to do with Resume Builder. I'm wondering if maybe you were planning to use it as a template? Next time it might be better to hold off pushing code that isn't ready?

Reply:
Nothing ... It was a copy and paste from another project. I should have deleted it. Good catch. Thanks